unittests/eosio_system_tester.hpp

```diff
@@ -538,7 +538,8 @@ class eosio_system_tester : public T {
             BOOST_REQUIRE_EQUAL( T::success(), regproducer(p) );
          }
       }
-      T::produce_blocks( 250);
+      T::produce_block();
+      T::produce_block(fc::seconds(1000));
```

Comment:
Why `1000`? I would have used `125`, but maybe it doesn't matter?

Reply:
125 would have been fine. Doesn't matter as the performance is the same.